hudi-flink-datasource/hudi-flink/src/main/java/org/apache/hudi/table/catalog/HoodieCatalogUtil.java

```diff
@@ -61,9 +61,10 @@ public class HoodieCatalogUtil {
    * @param hiveConfDir Hive conf directory path.
    * @return A HiveConf instance.
    */
-  public static HiveConf createHiveConf(@Nullable String hiveConfDir) {
+  public static HiveConf createHiveConf(@Nullable String hiveConfDir, @Nullable org.apache.flink.configuration.Configuration flinkConf) {
```

Comment:
The method has only 1 caller, just make the param as `org.apache.flink.configuration.Configuration` should be fine.